vassal-app/src/main/java/VASSAL/build/module/ChessClockControl.java

```diff
@@ -563,6 +585,79 @@ public Command startNextClock() {
     return command;
   }
 
+  /**
+   * @return a command to start the opponents clock after stopping the one currently running (if any)
+   */
+  public Command startOpponentClock() {
+    Command command = new NullCommand();
+    final String mySide = PlayerRoster.getMySide();
+    System.out.println(mySide);
+
+    // If we don't have any clocks, can't start one.
+    // If there are not exactly 2 clocks, this function is invalid
+    // If player doesn't belong to a side, they don't have an opponent to send to
+    if (chessclocks.isEmpty() || chessclocks.size() != 2 || mySide == null || mySide.contains("observer")) {
+      return command;
+    }
+
+    Boolean clockTicking = false;
```

Comment:
`boolean` should always be used in preference to `Boolean` if no object is needed.